sw/airborne/firmwares/rotorcraft/autopilot_guided.h

```diff
@@ -96,9 +96,17 @@ extern void autopilot_guided_update(uint8_t flags, float x, float y, float z, fl
 #define GUIDED_FLAG_Z_OFFSET    (1<<2)
 #define GUIDED_FLAG_YAW_OFFSET  (1<<3)
 #define GUIDED_FLAG_XY_VEL      (1<<5)
-#define GUIDED_FLAG_2_VEL       (1<<6)
+#define GUIDED_FLAG_Z_VEL       (1<<6)
 #define GUIDED_FLAG_YAW_RATE    (1<<7)
 
 
+// Useful Combination of the flags fir the autopilot_guided_update
```

Comment:
I'm not so found of these defines, as everybody might want its own combination in the end. And even if it is shorter, it is not always easier to read.
If you want them for your flight plan, you can have these lines in the `header` section.